op-challenger/game/scheduler/coordinator.go

```diff
@@ -109,9 +114,12 @@ func (c *coordinator) createJob(game common.Address) (*job, error) {
 			return nil, fmt.Errorf("failed to create game player: %w", err)
 		}
 		state.player = player
+		state.status = player.Status()
 	}
 	state.inflight = true
-	return &job{addr: game, player: state.player}, nil
```

Comment:
Why don't we still return `&job{...}` here? We don't use it or modify it beyond setting fields.